lib/Dialect/HW/HWOps.cpp

```diff
@@ -730,22 +730,6 @@ static LogicalResult verifyInstanceOp(InstanceOp op) {
     return op.emitError("Cannot find module definition '")
            << op.moduleName() << "'";
 
-  if (auto paramDictOpt = op.parameters()) {
```

Comment:
Why does this check happen so early? Do we really care what the Verilog emitter supports at this point?

Reply:
More to the point, we shouldn't add `VerbatimParameterAttr` here since that would introduce a dependency to the `hw` dialect on the `sv` dialect.